src/scancode/resource.py

```diff
@@ -395,8 +432,8 @@ def _populate(self):
         def err(_error):
             """os.walk error handler"""
             self.errors.append(
-                'ERROR: cannot populate codebase: {}\n'.format(_error)
-                + traceback.format_exc())
+                ('ERROR: cannot populate codebase: %(_error)r\n' % _error)
```

Comment:
Was there a problem with using format here? Did you had an issue from mot having a repr?

Reply:
My bad. That actually resulted from a mistake while solving a merge conflict. I have reverted it back to the `format` version.